src/test/resources/test-application.yaml

```diff
@@ -0,0 +1,10 @@
+spring:
+  jpa:
+    show-sql: true
+    hibernate:
+      ddl-auto: update
+  datasource:
```

Comment:
remove this, please.
It would take resources from common context

Reply:
this will be good :

spring:
  jpa:
    show-sql: true
    hibernate:
      ddl-auto: validate
    open-in-view: false
  profiles:
    active: test

Reply:
done